cert/lib/cert/runner.rb

```diff
@@ -160,11 +160,18 @@ def create_certificate
       request_path = File.expand_path(File.join(Cert.config[:output_path], "#{certificate.id}.certSigningRequest"))
       File.write(request_path, csr.to_pem)
 
-      private_key_path = File.expand_path(File.join(Cert.config[:output_path], "#{certificate.id}.p12"))
+      private_key_path = File.expand_path(File.join(Cert.config[:output_path], "#{certificate.id}.pkey"))
       File.write(private_key_path, pkey)
 
       cert_path = store_certificate(certificate)
 
+      if Cert.config[:generate_p12]
+        p12_cert_path = File.join(Cert.config[:output_path], "#{certificate.id}.p12")
+        p12 = OpenSSL::PKCS12.create(Cert.config[:p12_password], type_name, pkey, certificate.download)
+        File.write(p12_cert_path, p12.to_der)
+        UI.message("p12 certificate: ".green + Pathname.new(p12_cert_path).realpath.to_s)
```

Comment:
Usually we use `File.expand_path` instead of `Pathname.new(...).realpath.to_s`. Unless that does something I don't know about?